packages/inter-protocol/src/proposals/addAssetToVault.js

```diff
@@ -204,7 +214,65 @@ export const registerScaledPriceAuthority = async (
   );
 };
 
-/** @typedef {import('./econ-behaviors.js').EconomyBootstrapPowers} EconomyBootstrapPowers */
+// wait a short while after end to allow things to settle
```

Comment:
Please make clear that this code is to work around https://github.com/Agoric/agoric-sdk/issues/8307 and that once that's solved in production we can remove this complication.

For example,

```suggestion
//#region timing robustness
// UNTIL https://github.com/Agoric/agoric-sdk/issues/8307
```

Reply:
done